js/app/modules/card/knowledgeDocument.js

```diff
@@ -237,7 +247,7 @@ const KnowledgeDocument = new Module.Class({
     _get_webview: function () {
         let webview = this._create_webview();
 
-        webview.renderer.enable_scroll_manager = this.show_toc;
+        webview.renderer.enable_scroll_manager = true;
```

Comment:
To summarize our discussions from today, I am convinced now that we should use the scroll manager. When you click on the link, it's not clear that you're going to remain in the same page, and so the scroll animation helps make it clear that you are elsewhere in the same document.